bigquery/routine_integration_test.go

```diff
@@ -88,6 +92,133 @@ func TestIntegration_RoutineJSUDF(t *testing.T) {
 	}
 }
 
+func TestIntegration_RoutineRemoteUDF(t *testing.T) {
+	if client == nil {
+		t.Skip("Integration tests skipped")
+	}
+	ctx := context.Background()
+
+	parentLocation := fmt.Sprintf("projects/%s/locations/%s", dataset.ProjectID, "us-central1")
```

Comment:
maybe give this a unique name like functionLocation?  The first read was confusing because of how this gets re-used to specify the connection location as well.